recipes/starshot_editoria11y/recipe.yml

```diff
@@ -0,0 +1,10 @@
+name: Editoria11y support
```

Comment:
I think that we want a little bit more generic of a name here. I propose `starshot_accessibility_tools`  as the machine name instead, to match the proposed track name in https://www.drupal.org/project/starshot/issues/3454529. It also gives us a little wiggle room to add _other_ tools here that aren't Editoria11y. (And the human-readable name should be "Accessibility tools" or similar.)